cmd/policies.go

```diff
@@ -309,3 +325,345 @@ func viewPolicyCmd(ctx *cli.Context) error {
 
 	return nil
 }
+
+const policyTestFailed = "Could not test policy."
+
+var permissionString = map[bool]string{
+	true:  "yes",
+	false: "no",
+}
+
+func testPolicies(ctx *cli.Context) error {
+	cfg, err := config.LoadConfig()
+	if err != nil {
+		return err
+	}
+
+	action, userName, path, err := parseArgs(ctx)
+	if err != nil {
+		return err
+	}
+
+	client := api.NewClient(cfg)
+	c := context.Background()
```

Comment:
This context should be shared down through all the funcs; their usage should be hierarchical (though in this case, just one master one will do).